projects/packages/forms/src/blocks/contact-form/variations.js

```diff
@@ -253,12 +261,19 @@ const variations = compact( [
 					label: __( 'Please rate our website', 'jetpack-forms' ),
 					required: true,
 					options: [
-						__( '1 - Very Bad', 'jetpack-forms' ),
+						__( '1 - Very bad', 'jetpack-forms' ),
```

Comment:
Unrelated; we're favoring sentence case instead of title case.

Reply:
In the edit function of the block `options` are only used for a `has-placeholder` class name. I don't think they are needed here and could be removed.